gallery_dl/extractor/twitter.py

```diff
@@ -231,7 +231,7 @@ def _tweets_from_api(self, url, max_position=None):
                     data["items_html"], '<div class="tweet ', '\n</li>'):
                 yield tweet
 
-            if not data["has_more_items"]:
+            if not data["has_more_items"] and data["min_position"] == None:
```

Comment:
`data` is not guaranteed to contain a `min_position` key (https://github.com/mikf/gallery-dl/issues/544#issuecomment-573416517), so either check with `data.get("min_position") is None` or `("min_position" in data and data["min_position"] is None)`, depending on whether there is a difference between *`min_position` is set and `None`* and *`min_position` is not set*.

(`is` is also preferred over `==` when testing for `None`)

Reply:
I didn't know that, whoops. The fix ended up being a bit more complicated then I thought. There is 6 possible responses from Twitter:

gallery-dl should continue:
`{"has_more_items":True}`     (This happens on the first page)
`{"has_more_items":True, "min_position":"123..."}`
`{"has_more_items":False, "min_position":"123..."}`

gallery-dl should stop:
`{"has_more_items":False}`
`{"has_more_items":False, "min_position":None}`
`{"has_more_items":True, "min_position":None}`

Basically we want to stop whenever `min_position` is None or doesn't exist, except when there is more items and `min_position` doesn't exist.